content/src/main/content/jcr_root/apps/acs-commons/renderconditons/isfeatureenabled/isfeatureenabled.jsp

```diff
@@ -0,0 +1,33 @@
+<%
```

Comment:
I don't think this is needed. There is already `granite/ui/components/foundation/renderconditions/feature`